release_notes/41-release-notes.adoc

```diff
@@ -122,7 +122,7 @@ In {product-title-short} 4.1, you can enhance Central's security and login capab
 This feature enables you to use the `roxctl` CLI and integrate with authentication systems such as OpenID Connect (OIDC) for a seamless single sign-on (SSO) experience without requiring additional tokens or passwords.
 Previously, the `roxctl` CLI only supported token and password-based workflows.
 
-For more information, see xref:../cli/getting-started-cli.adoc#configuring-an-authentication-provider-in-central_cli-getting-started[Configuring an authentication provider in Central].
+For more information, see xref:../cli/using-the-roxctl-cli.adoc#configuring-an-authentication-provider-in-central_using-roxctl-cli[Configuring an authentication provider in Central].
```

Comment:
Can we not name this `Configuring an authentication provider`?

The documentation and steps outlined are not regarding _configuring_ an auth provider, but _using_ an auth provider to authenticate with roxctl against Central. The title and content is misleading to me, so let's stick to `Using an authentication provider to authenticate with roxctl`?